### PR TITLE
Add midnight css desktop

### PR DESCRIPTION
**Pull request by AbuTalha3**
In this PR I have:-
- Added the desktop version to my portfolio project.
- Worked with display block and none.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,14 +12,19 @@
...
             <a class="logo" href="#">ABU TALHA</a>
             <nav class="menu-icon">
-                <img class="menubars" src="images/menu.png" alt="Menu"> 
+                <img class="menubars" src="images/menu.png" alt="Menu">
+                <ul class="heading-menu">
+                  <li>Portfolio</li>
+                  <li>About</li>
+                  <li>Contact</li>
+              </ul> 
             </nav>
```
Comment (emmiiorji) [inline index.html:21]: Kindly adjust the design of the navigation bar to match the design.
Points to note:
- There's more margin to the left of the logo and the right of the contact nav
- A look at the part of the design shown below shows that the navigation bar is fixed. Kindly implement this too.
- There some shadow showing the bottom of the nav bar

Design:
image

Your nav bar:
image

File: index.html [hunk 2]
```diff
@@ -12,14 +12,19 @@
...
         </header>
         <section class="homepage">
             <div class="intro">
                 <h1>
                     I'm Abu Talha. <br>  
-                    Glad to see <br> you!
+                    Glad to see <br class="break"> you!
                 </h1>                                  
             </div>
```
Comment (emmiiorji) [inline index.html]: Kindly adjust the hero section to match the design. The items below are vertically aligned to the left.

Your design:
image


Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @AbuTalha3, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Good design
- Proper documentation in readme
- Linter checks pass


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -468,3 +502,249 @@ body {
...
+    background: url(./images/Aboutme-Big-screen.png) no-repeat;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 12px;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
```
Comment (IngDavidTM) [inline style.css]: - [ ] Although you successfully implemented the necessary changes with excellence :+1: , as previously mentioned, please ensure that the components on the homepage are properly aligned.


https://github.com/AbuTalha3/Portfolio-Setup/assets/105117832/1edc6a67-b2eb-45b9-bfe5-2f71d1c88a69

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @AbuTalha3 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -468,3 +502,242 @@ body {
...
+    background-size: contain;
+    background-repeat: no-repeat;
+    padding: 359px 260px;
+  }
+
+  .intro {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```
Comment (tugsem) [inline style.css:547]: - [ ] I think some of the designs are broken after the reviews, firstly, please add more padding to this class as it sticks to the top of the 768px screens. 
image

File: style.css [hunk 5]
```diff
@@ -468,3 +502,242 @@ body {
...
+
+  .cards {
+    display: grid;
+    padding: 16px;
+    grid-template-columns: repeat(2, 1fr);
+    justify-content: center;
+    place-items: center;
+    gap: 20px;
+    background-color: #e5e5e5;
+  }
```
Comment (tugsem) [inline style.css]: - [ ] Please align the cards according to the figma design. Your cards seem like this on 768px screens: 
image
figma: 
image

File: style.css [hunk 6]
```diff
@@ -468,3 +502,242 @@ body {
...
+
+  .form-section {
+    background-image: url(./images/Aboutme-Big-screen.png);
+    background-position: center;
+    background-size: 80%;
+    background-repeat: no-repeat;
+    background-color: #4949de;
+    justify-content: center;
+    align-items: center;
```
Comment (tugsem) [inline style.css]: - [ ] (OPTIONAL) Please apply the right background image as it is overlapping the text and makes it hard to read:
image
figma:
image

File: style.css [hunk 7]
```diff
@@ -468,3 +502,242 @@ body {
     grid-template-columns: repeat(3, 1fr);
   }
 }
+
+@media screen and (min-width: 768px) {
+  header {
```
Comment (tugsem) [inline style.css:507]: - [ ] As the first reviewer required, please add a shadow under your header and make it fixed to match the figma design. So when we scroll down the header always be on top for us to navigate:
image

**Review by tugsem — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️
Hello @AbuTalha3 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Highlights 🚀
✔️Good job on fixing the alignment of the hero section

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ofarouq310 — APPROVED** [review]
Hi @AbuTalha3, ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Followed GitHub flow.
✅ No linters errors.
✅ Descriptive PR title & summary.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**